src/components/@molecules/SearchInput/SearchInput.tsx

```diff
@@ -338,6 +341,8 @@ const createSearchHandler =
       { lastAccessed: Date.now(), nameType, text, isValid: selectedItem.isValid },
     ])
 
+    trackRegistrationEvent('Search selected', { keyword: text })
```

Comment:
Since existing event names are lower cased (`register/commit/renew`) shall we do all snake case? `search_selected` not `Search selected` ?

Reply:
@makoto  I've switched to snake case. Please review the changes, thank you.